.github/workflows/build-example-desktop-mac.yml

```
@@ -0,0 +1,41 @@
+name: Build Example Desktop (macOS)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)